mirage-net-unix.opam

```diff
@@ -1,30 +1,31 @@
-opam-version: "1.2"
-maintainer:   "anil@recoil.org"
-authors:      [
-  "Anil Madhavapeddy"
-  "David Scott"
-  "Thomas Gazagnaire"
-  "Hannes Mehnert"
+opam-version: "2.0"
+maintainer: "anil@recoil.org"
+authors: [
+  "Anil Madhavapeddy" "David Scott" "Thomas Gazagnaire" "Hannes Mehnert"
 ]
-homepage:    "https://github.com/mirage/mirage-net-unix"
+license: "ISC"
+homepage: "https://github.com/mirage/mirage-net-unix"
+doc: "https://mirage.github.io/mirage-net-unix/"
 bug-reports: "https://github.com/mirage/mirage-net-unix/issues"
-dev-repo:    "https://github.com/mirage/mirage-net-unix.git"
-license:     "ISC"
-doc:         "https://mirage.github.io/mirage-net-unix/"
-
-build: [
-  [ "jbuilder" "subst"] {pinned}
-  [ "jbuilder" "build" "-p" name "-j" jobs ]
-]
-
 depends: [
-  "jbuilder"  {build & >="1.0+beta9"}
+  "ocaml" {>= "4.04.2"}
+  "dune" {build & >= "1.0"}
   "cstruct" {>= "1.7.1"}
   "cstruct-lwt"
   "lwt" {>= "2.4.3"}
   "mirage-net-lwt" {>= "1.0.0"}
   "io-page-unix" {>= "2.0.0"}
   "tuntap" {>= "1.3.0"}
-  "alcotest" {test}
+  "alcotest" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
```

Comment:
`dune runtest` missing?

Reply:
deliberately left out in this case -- the tests need elevated privileges to access tun on some platforms, so best run manually